### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by tGodson**

### CODE REVIEW

**Comment by sumancrest0001** [comment]
### STATUS: Require changes.
Awesome work TEAM :clap: :clap: 

**The page layout looks awesome.**

[Require] We have to fix one validation error in the HTML file.
Screenshot from 2020-04-04 21-13-57

**Comment by sumancrest0001** [comment]
[REQUIRED] You have a lot of repeated margin-left, margin-right in your CSS file. You can write one line code for that `margin: top right bottom left`. You can also define a class for `font-family: nyt-franklin, arial, helvetica, sans-serif;` and add that class where ever it is required. These steps to keep your code tidy.

**Comment by sumancrest0001** [comment]
[Required] :confused: :confused: Can you please tell us why you used **node modules**? If you want to use it, please add it to the .gitignore file and push the project to GitHub.

**Comment by sumancrest0001** [comment]
[REQUIRED] We have to also fix a couple of issues related to the layout of the page. Please make the following changes. After these changes, it will look similar to that of the original site.

Please add Editor's pick section beside the main section. [Requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/aside_requirement.png)

In the original site :point_down:  :point_down: 
Screenshot from 2020-04-04 21-56-05

**Comment by sumancrest0001** [comment]
The main logo should be a link not only an image. 
Screenshot from 2020-04-04 21-30-34

**Comment by sumancrest0001** [comment]
Text of **The Theory of Inflation** section is not aligned as that of the original one.

**In your site.** :point_down:  :point_down: 
Screenshot from 2020-04-04 21-37-46

Should be :point_down: 
Screenshot from 2020-04-04 21-37-58

**Comment by sumancrest0001** [comment]
Social media and other icons are not visible. Please check them out and fix them.
Screenshot from 2020-04-04 21-30-49

**Comment by sumancrest0001** [comment]
Since this project focus on the alignment of items and positioning, we have to make sure that the positioning of each element is perfect. The Footer section looks a little bit different from the original one. Please make the following changes.

in your project :point_down: :point_down: 
Screenshot from 2020-04-04 21-34-51


Should be :point_down: 
Screenshot from 2020-04-04 21-35-02

**Comment by sumancrest0001** [comment]
Fix these issues. :point_down: :point_down: 
in your project :point_down:  :point_down: 

Screenshot from 2020-04-04 21-35-15
should be :point_down: 
Screenshot from 2020-04-04 21-35-32

[TIPS] Reduce space between elements.

**Comment by sumancrest0001** [comment]
Align the small texts from right not left.
**In your project.**
Screenshot from 2020-04-04 21-33-35

Should be :point_down: 
Screenshot from 2020-04-04 21-33-48

**Comment by sumancrest0001** [comment]
There is a footer section that should be sticky at the bottom all the time. It is missing in your project. 
**In the original project.**

Screenshot from 2020-04-04 22-12-36

**Comment by sumancrest0001** [comment]
**Kindly make the above changes and submit them for another review. :pray: :pray: :pray:**

Need help? Reach out on slack handle **Suman Shrestha**

**Comment by sumancrest0001** [comment]
### STATUS: changes required

You have done nice improvements :+1: :+1: :clap: 
But there are still a couple of things that need to be fixed as suggested in the previous review.

**Please fix the following issues. :point_down:**

File: index.html [hunk 1]
```diff
@@ -0,0 +1,671 @@
...
+<body>
+    <header id="header">
+        <nav class="left-nav">
+            <img src="./img/menu.png" alt="menu" />
+            <img src="./img/search.png" alt="search" />
+            <a class="space-cosmos">Space & Cosmos</a>
+        </nav>
+        <nav>
+            <img src="./img/logo.PNG" alt="logo" />
+        </nav>
```
Comment (sumancrest0001) [inline index.html]: [REQUIRED] Please add <a> tag on the logo image. It should be link also not only an image.

**Comment by sumancrest0001** [comment]
[REQUIRED] Please .gitignore file and add node modules on that so that It is not pushed to remote repository. It makes your repo bulky.

**Comment by sumancrest0001** [comment]
[Required] You have added another footer section which is fine but we want you to add a sticky footer section as stated in the previous review. Please have a close look.

**In the original site :point_down:  :point_down:** 
Screenshot from 2020-04-05 09-39-04

the last section is a sticky footer. [reference](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)

**Comment by sumancrest0001** [comment]
[Required] You have fixed text alignment issues with the footer section but there is an overlap between contents in the subscribe section.
Have a look :point_down: 
Screenshot from 2020-04-05 09-41-50

**Comment by sumancrest0001** [comment]
[Required] Social media icons are still not visible. Please fix this issue also.
Screenshot from 2020-04-05 09-45-25

**Comment by sumancrest0001** [comment]
[Important change] In the HTML file you have used font-family as class`<p class="font-family">By LARRY BUCHANAN and JONATHAN CORUM</p>` but defined as **id** in the CSS file  `#font-family {
  font-family: nyt-franklin, arial, helvetica, sans-serif;
}`

**Comment by sumancrest0001** [comment]
**Kindly make the above changes and resubmit for another review. :+1: :+1: 
Happy coding.**

**Comment by tGodson** [comment]
the social media icons are displaying on my screen

**Review by christianotieno — COMMENTED** [review]
## Pull Request under review by Christian Otieno (TSE)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# Positioning and Floating Elements
+This project consisted of making a clone of a New York Times article page.The New York Times has been a leader in experimenting with the inclusion of multimedia content in traditional stories. Which creates all kinds of interesting position questions.
+Here's the original page http://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0
+
+![screenshot](./form-screenshot.png)
+
```
Comment (christianotieno) [inline README.md]: ## [REQUIRED ]- FIX - The README
Your README should match the [standards](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/) that have been set in the pathwright. It is among the [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) for your repo to get approved.
##
- For some reason, most of your images are not being displayed properly. An example can be found in your README for instance.
README

I presume you meant this: 👇 ?
```suggestion
![screenshot](./app_screenshot.png)
```


This and among the many, my colleague has highlighted in the previous changes as well. What they are seeing is exactly what I seeing on my end. They are working perfectly well in localhost but not in production.

Comment (christianotieno) [reply README.md]: We also don't have a live link to the site in your README 
- You can use [githack](https://raw.githack.com/) to set up a live link and embed it in your README.

Comment (christianotieno) [reply README.md]: ## [REQUIRED] - FIX - The positioning of the More in Space and Astronomy section
The 'More in Space and Astronomy' section should be fixed to match the original version;
- This is what we currently have: 👇 
position_clone
 - THis is what we should be aiming to achieve: 👇 
position
Resize it.

Comment (christianotieno) [reply README.md]: ## [REQUIRED] - FIX - The footer
We should also fix the footer as well.
- This is what we have currently: 👇
footer_clone
- We should aim to have it look something close to this: 👇 
footer

Comment (christianotieno) [reply README.md]: ## [OPTIONAL] - REPLACE - The emended image and video.
I applaud the good job done into building this project.
- We can always try and improve on some areas that might make our site even more appealing.
- Consider these changes optional as the general minimum requirements have already been met in these sections. 
##
Our embedded video: 👇 
stanford_clone
We may change it to match the original: 👇 
stanford_professor
Our embedded image: 👇 
bicept_clone

We may change it to match the original: 👇 
bicept

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @tGodson & @imhilla   :wave:
-
You have done a splendid job! :tada: Keep it up.
We, however, do have some few changes to carry out before we proceed (see review comment).
##
In addition to my colleague's previous changes, below are some small issues I found while reviewing your awesome project.

Please makes changes and submit another review request.
- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- Again when you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Comment by uryelah** [comment]
Hello **Hillary Odhiambo** & **Godson Tendongze Zeingkeang**, I'm Sarah, your last reviewer for this PR 👩‍🚀  

Great job working on all the fixes pointed out by the last TSE! 

[IMPORTANT] Sorry that this was not noted before but remember to add, and to keep, a descriptive Pull request message and title. Like the one in the gif below:
69548761-3a8d3600-0f98-11ea-9f19-14da361c12e2

I'll mark this as optional since it was not noted in the previous review, but please remember to add the PR message in your next projects.

------
## Your project is approved! 🎊 🥅 💛 
### 
**You may close this Pull Request and merge the changes**

----

### Optional Changes

- [ ] Add a descriptive PR message.

- [ ] The navbar and fixed footer should take 100% of the screen width. Sometimes you can miss that according to your screen size. A fast trick to see how your page looks in a slightly wider screen is clicking `ctrl+`, or you could use your browser responsiveness tools.

----

**Keep being awesome devs and good luck with your next project!**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Pull request merged**